src/backend/aspen/app/views/usergroup.py

```diff
@@ -43,3 +51,28 @@ def usergroup():
             # all fields have updated successfully
             db_session.flush()
             return jsonify(success=True)
+
+        if request.method == "POST":
+            if user.system_admin:
+                # check we're only getting fields that we expect
+                new_user_data: Dict[str, Union[str, bool]] = {
+                    k: v for k, v in request.get_json().items() if k in POST_USER_FIELDS
+                }
+                missing_required_fields = [
+                    f for f in POST_USER_FIELDS if f not in new_user_data.keys()
+                ]
+                if missing_required_fields:
+                    return Response(
+                        f"Insufficient information required to create new user, {missing_required_fields} are required",
+                        400,
+                    )
+                else:
+                    user = User(**new_user_data)
+                    db_session.add(user)
+                    db_session.flush()
+                    return jsonify(success=True)
```

Comment:
Do we want to interface with Auth0 to create the user there?

Reply:
that would be fancy! I wasn't sure if that was what we wanted here or not @shanaxel42 what do you think? Was that part of the AC for this?

Reply:
FWIW, the auth0 management credentials are in `aspen-config` so you could theoretically do this.

Reply:
yes i can definitely do this, do you think i should do that in this PR? or should i merge this and then add that functionality in another PR? I think it might be cleaner to add it in a follow up PR

Reply:
Subsequent PR makes sense.